tv_apps/fling-player/test/unit/master_debug_test.js

```diff
@@ -0,0 +1,115 @@
+/* global mDBG */
+'use strict';
+
+requireApp('fling-player/js/master_debug.js');
+
+suite('fling-player/MasterDebug', function() {
+
+  var exp, mockWindow;
+
+  setup(function () {
+    mockWindow = {};
+    mockWindow.log = mockWindow.warn = mockWindow.error = function () {};
+  });
+
+  teardown(function () {
+    exp = undefined;
+  });
+
+  test('should turn off debug mode before push to moz upstream', function () {
+    assert.isFalse(mDBG.isDBG(),
+      'Please turn off debug mode before push to moz upstream'
+    );
+  });
+
+  test('should set debug flag', function () {
+
+    exp = true;
+    mDBG.setDBG(exp);
+    assert.equal(mDBG.isDBG(), exp);
+
+    exp = false;
+    mDBG.setDBG(exp);
+    assert.equal(mDBG.isDBG(), exp);
+  });
+
+  suite('behavior in the debug mode', function () {
+
+    var a = 'a', b= 'b';
```

Comment:
and I would suggest to declare them separately when they have default value assignment.